src/engine/Uno.Build/Stuff/Disk.cs

```diff
@@ -91,7 +91,10 @@ public static void TouchFile(Log log, string name)
         {
             try
             {
-                File.SetLastWriteTimeUtc(name, DateTime.UtcNow);
+                if (!File.Exists(name))
+                    File.Create(name);
+                else
+                    File.SetLastWriteTimeUtc(name, DateTime.UtcNow);
```

Comment:
Technically speaking, I think this is still racy, as groups of filesystem operations aren't atomic. But I doubt it'll ever matter.

Reply:
If two processes ends up here at the same time, one of the processes might throw but that's fine since the exception is ignored and the file should get it's timestamp updated by one of the processes, which is what we want.